apps/exports/decorators.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.decorators import user_passes_test
+from django.core.exceptions import PermissionDenied
+
+
+def staff_required(function=None, login_url=None, raise_exception=True):
```

Comment:
Is this necessary at all? I think [user_passes_test](https://docs.djangoproject.com/en/3.1/topics/auth/default/#django.contrib.auth.decorators.user_passes_test) already handles the login/authentication aspect for you.